src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/ConvertToBase64Fuzzer.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+
+namespace DotnetFuzzing.Fuzzers
+{
+    internal sealed class ConvertToBase64Fuzzer : IFuzzer
```

Comment:
This isn't covering that much logic, and the interesting inputs are the same as with `Base64Fuzzer`. Can we move the extra asserts into that one instead?

Reply:
Only Base64FormattingOptions.None path uses Base64 encoder, it has its own logic for other cases and also has logic for widening encoded char after calling Base64.EncodeTo***(...). 

i.e there is some logic we want to be cover, plus I assume other Conver APIs fuzz tests will be added to this one

Reply:
But the inputs are base64, so the interesting corner cases will be similar between base64 APIs on Convert, or on the Base64 class.

If we were to add more fuzz testing for other APIs on Convert, I don't think we'd want them to share the fuzz target with this one as the inputs for base64 parsing have nothing in common with interesting inputs for parsing an int32.

Reply:
Done